### PR TITLE
add logic,add ui components

### PR DESCRIPTION
**Pull request by VictorYarosh**

### CODE REVIEW

File: src/componets/ui/content-title/index.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import React from "react";
+
+import {
+  Column,
+  Title,
+  TitleDescription,
+  TitleWrapper,
+} from "../../ui/content-title/content-title.styled";
+
+const ContentTitle = () => {
```
Comment (toserg) [inline src/componets/ui/content-title/index.tsx:11]: export const

File: src/componets/ui/content-title/index.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+              suggestion. 😉
+            </TitleDescription>
+          </TitleWrapper>
+        </Column>
+      </Row>
+    </>
+  );
+};
+
+export default ContentTitle;
```
Comment (toserg) [inline src/componets/ui/content-title/index.tsx:30]: remove

File: src/componets/ui/movies-cards/index.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+            height={400}
+            src={`https://image.tmdb.org/t/p/w500${data.results}`}
+          />
+          <CardMoviesTitle></CardMoviesTitle>
+        </CardMovies>
+      </CardMoviesWrapper>
+    </>
+  );
+};
+export default MoviesCards;
```
Comment (toserg) [inline src/componets/ui/movies-cards/index.tsx:38]: Move the component to the complex

**Review by toserg — APPROVED** [review]

File: src/componets/ui/content-title/index.tsx (hunk 1 above)
Comment (VictorYarosh) [reply src/componets/ui/content-title/index.tsx:11]: fixed this

File: src/componets/ui/content-title/index.tsx (hunk 2 above)
Comment (VictorYarosh) [reply src/componets/ui/content-title/index.tsx:30]: also fixed

File: src/componets/ui/movies-cards/index.tsx (hunk 3 above)
Comment (VictorYarosh) [reply src/componets/ui/movies-cards/index.tsx:38]: fixed

**Pull request merged**